### ISSUE TITLE
#include does not handle absolute paths

### ISSUE THREAD

**Issue by poppolopoppo** [fastbuild/fastbuild#48]
`BFFParser::ParseIncludeDirective` always append the path of the currrent file, assuming that given string is a relative path.

Ex :
````cpp
#include "c:\test.bff"
````
````
c:\test.bff(1,11): FASTBuild Error #1032 - Unable to open include 'c:\c:\test.bff'.
````

**Comment by ffulin** [comment]
I have fixed this in the Perforce mainline. The fix will be in the next release (v0.88). I'll leave this bug open until the next version is released.

**Comment by ffulin** [comment]
v0.88 has now been release and includes a fix for this issue.
